### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader:The name of the page is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9347]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#JAWS;#MAS1.3.1;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader:JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch screen reader as JAWS
8. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.

**Actual:**
When the page is loaded the "alert"  message is announced and the name of the screen in display is not announced.

Issue not  observed with NVDA

**Expected:**
The screen name in display should be announced.

**User Impact:**
Screen reader users will not know which page they have landed on.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EQc5q6gpkqZNpmI8Xdi7bQEBOZzBWTRK3Dag9l6FYcPQzg?e=h7NUKI

**Comment by alanrenmsft** [comment]
dupe of #9348

**Comment by Yashi1201** [comment]
As "SQL Server Big Data Cluster" feature has been removed
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
